BitList.cs

```diff
@@ -0,0 +1,1553 @@
+// By Lorenzo Lotti
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This file has 1065 lines, which is greater than 1000 authorized. Split it into smaller files.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=5559704)